addons/spreadsheet/static/src/assets_backend/spreadsheet_binary_field/spreadsheet_binary_field.js

```diff
@@ -0,0 +1,30 @@
+import { registry } from "@web/core/registry";
+import { WarningDialog } from "@web/core/errors/error_dialogs";
+import { _t } from "@web/core/l10n/translation";
+import { BinaryField, binaryField } from "@web/views/fields/binary/binary_field";
+import { useService } from "@web/core/utils/hooks";
+
+export class SpreadsheetBinaryField extends BinaryField {
```

Comment:
I guess we want to keep the field in the list to allow the upload of a json ? Sounds a bit strange to have a custom component that is only used in debug mode, but w/e.

The behaviour is kinda broken tho.
1) the download buttons are still here. It doesn't make sense to leave them so they can spawn a dialog error, we should remove or gray them out.
2) If you want to upload a JSON, you will first click on the field and ... it tries to dowload it and you get an error message. It should be possible to upload a file _without_ having an error in your face if we want to keep the field to upload a json.

You should talk with Francois to see exactly what we want :)

Reply:
Hello @hokolomopo 🙂 
Thanks for for your feedback! Sure, we'll discuss this with Francois!